### DISCUSSION TITLE
Station Name on Top Line to be centred on NR

### DISCUSSION THREAD

**Discussion by JoshX21A** [jfoot/Bus-Departure-Board#23] [General]
Is it possible to have the Top Line/Station Name to be centred on the screen?

**Comment by jfoot** [comment]
Hi, this is possiable yeah, I can implement this feature for you soon. I'll let you know once it's done :)

**Comment by jfoot** [comment]
Hi @JoshX21A, this has now been implemented in the latest version V2.6.NR, your display should automatically update to this over night, or you can force it to do so now using the command  "`git -C ./Bus-Departure-Board/ pull`".

By default it will now centre align both the "loc" and "date" header, but you can revert back to right alignment using the new `--HeaderAlignment` tag.
